service-templates/README.md

```diff
@@ -0,0 +1,32 @@
+This directory contains service (with plugins and UIs) templates that can be instantiated with `cargo generate`.
+
+# Install
+
+`cargo install cargo-generate`
+
+# Usage
+
+## Instantiating the template
+
+> Only for Rust projects so far
+
+Instanstiate a Rust service from Git:
+
+`cargo generate --git fractally/psibase/templates` rust
```

Comment:
I don't see how this can work.

Reply:
The other problem: our repo is gofractally

Reply:
Doh! Right, that's bad. Thanks for the safety tip Egon!
![image](https://github.com/user-attachments/assets/afffdfb6-ad1a-455d-8f7a-15d7e8ee8f47)